client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class DesktopDownloadCard extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.windowsLink = 'https://apps.wordpress.com/d/windows?ref=getapps';
+		this.macLink = 'https://apps.wordpress.com/d/osx?ref=getapps';
+		this.linuxTarLink = 'https://apps.wordpress.com/d/linux?ref=getapps';
+		this.linuxDebLink = 'https://apps.wordpress.com/d/linux-deb?ref=getapps';
+	}
```

Comment:
It's fine to keep these as static constants outside of the Component. If we intended for these values to change an alternative would be setting default state via:

```jsx
state = {
    windowsLink: 'https://apps.wordpress.com/d/windows?ref=getapps',
};
```